control/autoware_autonomous_emergency_braking/src/node.cpp

```diff
@@ -928,53 +932,21 @@ void AEB::getClosestObjectsOnPath(
   if (ego_path.size() < 2 || points_belonging_to_cluster_hulls->empty()) {
     return;
   }
-  const auto ego_is_driving_forward_opt = autoware::motion_utils::isDrivingForward(ego_path);
-  if (!ego_is_driving_forward_opt.has_value()) {
-    return;
-  }
-  const bool ego_is_driving_forward = ego_is_driving_forward_opt.value();
-  // select points inside the ego footprint path
-  const auto current_p = [&]() {
-    const auto & first_point_of_path = ego_path.front();
-    const auto & p = first_point_of_path.position;
-    return autoware::universe_utils::createPoint(p.x, p.y, p.z);
-  }();
 
+  const auto longitudinal_offset_opt = utils::getLongitudinalOffset(
+    ego_path, vehicle_info_.front_overhang_m, vehicle_info_.rear_overhang_m);
```

Comment:
```suggestion
  const auto longitudinal_offset_opt = utils::getLongitudinalOffset(
    ego_path, vehicle_info_.max_longitudinal_offset_m, vehicle_info_.rear_overhang_m);
```
Found the issue. Not sure what the front overhang is supposed to represent, but its value is 1 meter which makes no sense. We want the distance from base link to the front of the object

Reply:
Nice!

>Not sure what the front overhang is supposed to represent,

`front_overhang` is the distance between front of the vehicle and front wheel link as attached in the screenshot.

![image](https://github.com/user-attachments/assets/0d563232-d667-4196-b62f-3e5ae5e82d59)

Reply:
Thank you so much! addressed in https://github.com/autowarefoundation/autoware.universe/pull/9462/commits/5260283861e9c3f1d81432d1b7298561dbad4c20